airbyte-integrations/connectors/source-facebook-pages/build.gradle

```diff
@@ -0,0 +1,14 @@
+plugins {
+    id 'airbyte-python'
+    id 'airbyte-docker'
+    id 'airbyte-source-acceptance-test'
+}
+
+airbytePython {
+    moduleDirectory 'source_facebook_pages_singer'
+}
+
+dependencies {
+    implementation files(project(':airbyte-integrations:bases:source-acceptance-test').airbyteDocker.outputs)
+    implementation files(project(':airbyte-integrations:bases:base-python').airbyteDocker.outputs)
```

Comment:
this line (13th) is deprecated but still exists in template-generator, need to be removed (look on another connectors for example)